src/api/notify-namespace.ts

```diff
@@ -0,0 +1,538 @@
+    >(
+      this.config,
+      AlchemyApiType.WEBHOOK,
+      'team-webhooks',
+      'webhooks:getAll',
+      {},
+      {
+        headers: { 'X-Alchemy-Token': this.config.notifyAuthToken! }
+      }
+    );
+    return {
+      webhooks: parseRawWebhookResponse(response),
+      totalCount: response.data.length
+    };
+  }
+
+  /**
+   * Get all addresses tracked for the provided {@link AddressActivityWebhook}.
+   *
+   * @param addressWebhook The Address Activity webhook.
+   * @param options Pagination options when fetching addresses.
+   */
+  getAddresses(
+    addressWebhook: AddressActivityWebhook,
+    options?: GetAddressesOptions
+  ): Promise<AddressActivityResponse>;
+
+  /**
+   * Get all addresses tracked for the provided {@link AddressActivityWebhook}.
+   *
+   * @param webhookId The id of the address activity webhook. Passing in an
+   *   incorrect id will result in a response object with no addresses.
+   * @param options Pagination options when fetching addresses.
+   */
+  getAddresses(
+    webhookId: string,
+    options?: GetAddressesOptions
+  ): Promise<AddressActivityResponse>;
+  async getAddresses(
+    webhookOrId: AddressActivityWebhook | string,
+    options?: GetAddressesOptions
+  ): Promise<AddressActivityResponse> {
+    this.verifyConfig();
+    const webhookId =
+      typeof webhookOrId === 'string' ? webhookOrId : webhookOrId.id;
+    const response = await requestHttpWithBackoff<
+      {},
+      RawAddressActivityResponse
+    >(
+      this.config,
+      AlchemyApiType.WEBHOOK,
+      'webhook-addresses',
+      'webhooks:getAddresses',
+      {
+        webhook_id: webhookId,
+        limit: options?.limit,
+        after: options?.pageKey
+      },
+      {
+        headers: { 'X-Alchemy-Token': this.config.notifyAuthToken! }
+      }
+    );
+    return parseRawAddressActivityResponse(response);
+  }
+
+  /**
+   * Get all NFTs tracked for the provided {@link NftActivityWebhook}.
+   *
+   * @param nftWebhook The NFT Activity webhook.
+   * @param options Pagination options when fetching NFT filters.
+   */
+  getNftFilters(
+    nftWebhook: NftActivityWebhook,
+    options?: GetAddressesOptions
+  ): Promise<NftFiltersResponse>;
+
+  /**
+   * Get all addresses tracked for the provided {@link NftActivityWebhook}.
+   *
+   * @param webhookId The id of the NFT activity webhook. Passing in an
+   *   incorrect id will result in a response object with no filters.
+   * @param options Pagination options when fetching nft filters.
+   */
+  getNftFilters(
+    webhookId: string,
+    options?: GetAddressesOptions
+  ): Promise<NftFiltersResponse>;
+  async getNftFilters(
+    webhookOrId: NftActivityWebhook | string,
+    options?: GetAddressesOptions
+  ): Promise<NftFiltersResponse> {
+    this.verifyConfig();
+    const webhookId =
+      typeof webhookOrId === 'string' ? webhookOrId : webhookOrId.id;
+    const response = await requestHttpWithBackoff<{}, RawNftFiltersResponse>(
```

Comment:
These calls are frequent enough in this file that you might consider making a helper function that calls `requestHttpWithBackoff` but sets the options which are common in this class without having to pass them in repeatedly; that is `this.config`, `AlchemyApiType.WEBHOOK`, and the `X-Alchemy-Token` header. Not that important if something makes it annoying to do.

Reply:
Great suggestion, much cleaner!